x/denommetadata/keeper/grpc_query.go

```diff
@@ -25,16 +24,6 @@ func NewQuerier(k Keeper) Querier {
 	return Querier{Keeper: k}
 }
 
-// Params returns denommetadata module params
-func (q Querier) Params(
-	c context.Context,
-	_ *types.QueryParamsRequest,
-) (*types.QueryParamsResponse, error) {
-	ctx := sdk.UnwrapSDKContext(c)
-	params := q.GetParams(ctx)
-	return &types.QueryParamsResponse{Params: params}, nil
-}
-
 // IBCDenomByDenomTrace returns IBC denom base on denom trace
 func (q Querier) IBCDenomByDenomTrace(
```

Comment:
I know it's not related, but this should be covered by a test. Maybe leave a `TODO`, this PR is already big enough.